README.md

```diff
@@ -30,8 +30,14 @@ You'll also need to clone a new `.env` file from the `.env.template` to store lo
 $ cp .env.template .env  # (first time only)
 ```
 
+
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+
+Update the three trello vars in the .env file to include the correct board id, API key and API token.
```

Comment:
Thanks for updating this, it would be nice to also briefly mention how to generate/access these values for future users